MaterialLib/MPL/Utils/CheckVanGenuchtenExponentRange.cpp

```diff
@@ -0,0 +1,28 @@
+/**
+ * \file
+ * \copyright
+ * Copyright (c) 2012-2020, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ * Created on April 20, 2020, 10:47 AM
+ */
+
+#include "CheckVanGenuchtenExponentRange.h"
+
+#include "BaseLib/Error.h"
+
+namespace MaterialPropertyLib
+{
+void checkVanGenuchtenExponentRange(const double m)
+{
+    if (!(m > 0 && m < 1))
```

Comment:
IMHO the following version is more readable.
```suggestion
    if (m <= 0 || m >= 1))
```

Reply:
Changed.